Author: Daniel Leidert (dale) <daniel.leidert@wgdd.de>.
Description: There are some wrong references to the PUBID 
"-//OASIS//ELEMENTS DocBook XML Information Pool V4.x//EN", which should be
"-//OASIS//ELEMENTS DocBook Information Pool V4.x//EN". This patch fixes them.

<URL:https://sourceforge.net/tracker/?func=detail&atid=384107&aid=1679712&group_id=21935>

<URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.5-1>

# only apply to DB >= 4.2!
for i in `find . -type f` ; do \
		sed -e 's/ELEMENTS DocBook XML Information Pool/ELEMENTS DocBook Information Pool/g' < $i > $i.new ; \
		mv $i.new $i ; \
done
--- a/docbook-4.2/dbpoolx.mod
+++ b/docbook-4.2/dbpoolx.mod
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.2//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.2//EN"
      "dbpoolx.mod">
      %dbpool;
 
--- a/docbook-4.3/dbpoolx.mod
+++ b/docbook-4.3/dbpoolx.mod
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.3//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.3//EN"
      "dbpoolx.mod">
      %dbpool;
 
--- a/docbook-4.4/dbpoolx.mod
+++ b/docbook-4.4/dbpoolx.mod
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.4//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.4//EN"
      "dbpoolx.mod">
      %dbpool;
 
--- a/docbook-4.5/dbpoolx.mod
+++ b/docbook-4.5/dbpoolx.mod
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.5//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.5//EN"
      "dbpoolx.mod">
      %dbpool;
 
